Marlin/src/gcode/gcode_d.cpp

```diff
@@ -174,7 +174,27 @@
         for (int i = 10000; i--;) DELAY_US(1000UL);
         ENABLE_ISRS();
         SERIAL_ECHOLNPGM("FAILURE: Watchdog did not trigger board reset.");
+      } break;
+
+      #if ENABLED(POST_MORTEM_DEBUGGING)
+      case 451: // Trigger all kind of faults to test exception catcher
+      case 452:
+      case 453:
+      case 454: { // Trigger all kind of faults to test exception catcher
+        SERIAL_ECHOLNPGM("Disabling heaters");
+        thermalManager.disable_all_heaters();
+        delay(1000); // Allow time to print
+
+        switch (dcode) {
+          case 451: *(int*)0 = 451; break; // Write at bad address
```

Comment:
One gcode with a parameter would be enough, isnt?

Reply:
How do you get a parameter in the current function signature `void GcodeSuite::D(const int16_t dcode)` ?

Reply:
sample for some random g code:
```cpp
    if (parser.seen('I')) {
      uint8_t cnt = 0;
      g29_repetition_cnt = parser.has_value() ? parser.value_int() : 1;
```

You can use O1, O2, O3, O4, (O for option)... I guess... of F, for 'fault'...